README.md

```diff
@@ -79,6 +79,18 @@ To monitor additional hosts, simply relate the Dockerbeat subordinate
 
     juju add-relation dockerbeat:beats-host my-charm
 
+## DockerBeat Delivery
+
+This charm makes use of Resources. A juju 2.0 feature. When deploying this
+charm on a juju 2.0 enabled controller, the upstream ingensi dockerbeat
```

Comment:
Capital i? According to this: https://github.com/Ingensi it is a company and always seems to be capitalized in github.